src/applications/disability-benefits/all-claims/Form526EZApp.jsx

```diff
@@ -1,16 +1,14 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
-import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
+// import moment from 'moment';
```

Comment:
did you mean to leave this commented out?

Reply:
oops, yeah I should remove that